.github/workflows/ci-docker-image-validation.yml

```diff
@@ -0,0 +1,40 @@
+name: Docker Image Validation
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+        branches: 
+            - master
```

Comment:
Should the trigger conditions be consistent with existing build tasks?

https://github.com/jaegertracing/docker-protobuf/blob/master/.github/workflows/jaeger-dockerimage.yml

```yaml
on: 
  push:
    tags:
      - '**'
    branches:
      - master
  pull_request:
    paths: 
      - .github/workflows/jaeger-dockerimage.yml
      - Dockerfile
      - protoc-wrapper
```

Alternatively, `checkout` and `build proto` can be integrated directly into the original CI so that we can directly confirm a successful build before the image is released.

Reply:
added triggers same as the `jaeger-dockerimage.yml` in this workflow